python/Ryu.py

```diff
@@ -216,23 +243,21 @@ def changeSwitchForwardingState(self, dpid=None, header=None, actions=None, comm
             obj['in_port'] = None
             
         if(header.has_key('nw_src')):
+            if header['nw_src'].version == 6:
+                obj['dl_type'] = 34525
```

Comment:
Seems like there needs to be an else here???  OF 1.0 can't include IPv6 maybe Ryu is doing something special for us?  In any case I think it is safer to have an else so that we don't try and set the nw_src/dst for IPv6 in OF1.0